### PR TITLE
Chore: Enable NEURON_SETTINGS for mainnet

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Enable the new neuron details page UI for mainnet.

# Changes

* Set `ENABLE_NEURON_SETTINGS` to `true` for both staging and mainnet.

# Tests

* Fix tests.

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by lmuntaner** [comment]
@dskloetd please review

File: dfx.json [hunk 1]
```diff
@@ -209,7 +209,7 @@
         "STATIC_HOST": "https://icp0.io",
         "FEATURE_FLAGS": {
           "ENABLE_CKBTC": true,
-          "ENABLE_NEURON_SETTINGS": false,
+          "ENABLE_NEURON_SETTINGS": true,
```
Comment (dskloetd) [inline dfx.json:212]: Remove this line and rely on the default value?

File: dfx.json [hunk 2]
```diff
@@ -238,7 +238,7 @@
         "HOST": "https://nnsdapp.dfinity.network",
         "FEATURE_FLAGS": {
           "ENABLE_CKBTC": true,
-          "ENABLE_NEURON_SETTINGS": false,
+          "ENABLE_NEURON_SETTINGS": true,
```
Comment (dskloetd) [inline dfx.json:241]: remove?

**Review by dskloetd — APPROVED** [review]

File: dfx.json (hunk 1 above)
Comment (lmuntaner) [reply dfx.json:212]: I wasn't sure what I preferred. But sure. No strong opinion.

**Pull request merged**